types/react-native/index.d.ts

```diff
@@ -3,6 +3,7 @@
 // Definitions by: Eloy Durán <https://github.com/alloy>
 //                 HuHuanming <https://github.com/huhuanming>
 //                 Kyle Roach <https://github.com/iRoachie>
+//                 Simon Knott <https://github.com/skn0tt>
```

Comment:
Are you aware that this means to sign up for future PR reviews?

Reply:
Yep. I'd be happy to help the TS community! :)